api/envoy/extensions/filters/network/geoip/v3/geoip.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.network.geoip.v3;
+
+import "envoy/config/core/v3/extension.proto";
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.network.geoip.v3";
+option java_outer_classname = "GeoipProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/geoip/v3;geoipv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: Geoip]
+// Geoip :ref:`configuration overview <config_network_filters_geoip>`.
+// [#extension: envoy.filters.network.geoip]
+
+message Geoip {
+  // The key to use for storing geolocation data in filter state.
+  // The geo data is stored as a typed ``GeoipInfo`` object which supports
+  // serialization for access logging and field-level access.
+  // If not specified, defaults to ``envoy.geoip``.
+  string metadata_namespace = 1;
```

Comment:
Because this is the filter state key, then I think this should not be `metadata_namespace` but filter_state_key or something? Although I think in most of case, we prefer to use the well-known name for filter state. That's same we needn't any additional configuration for this. We can use the `envoy.geoip` or `envoy.network.geoip` directly and document it clearly in the `docs/root/configuration/advanced/well_known_filter_state.rst`

Reply:
I'll make it WKT and remove this field. Thanks!

Reply:
Discussed offline and changed it to `envoy.geoip`.